cmd/stack/main.go

```diff
@@ -50,7 +53,13 @@ func main() {
 	)
 	cmd := kingpin.MustParse(app.Parse(os.Args[1:]))
 
-	zl := runtimelog.ZapLogger(*debug)
+	o := func(o *runtimezap.Options) {
+		o.Development = *debug
+		lvl := zap.NewAtomicLevelAt(zap.FatalLevel)
+		o.StacktraceLevel = &lvl
+	}
+	zl := runtimezap.New(o)
```

Comment:
i sort of like the idea of having these common options, that will get reused across all the controller based stacks we're building right now, in a common and reusable place like crossplane-runtime.  Then these consumers, such as stack-gcp/stack-aws/stack-azure would just call the single common function from crossplane-runtime to get an initialized log instance with a single line.

I see you started this common effort with https://github.com/crossplaneio/crossplane-runtime/pull/47.  Perhaps we can continue pursuing that?